packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -84,96 +317,14 @@ def create_client(
     MEASUREMENT_SERVICE_CLASS: Accepts one or more measurement service classes.
     Separate each service class with a space.
```

Comment:
with the latest update, this needs to be updated as we should give -s for each service class name.